test/cli_test.js

```diff
@@ -0,0 +1,88 @@
+        test.strictEqual(cli.program.usage(), '[options] [files...]', 'should tell how to use Bootlint.');
+        test.done();
+    },
+    'Disabled options': function (test) {
+        test.expect(5);
+        test.strictEqual(cli.program.option('')._allowUnknownOption, false, 'Don\'t allow unknown command line options.');
+        test.strictEqual(cli.program.option('')._name, 'grunt', 'should list the program\'s name.');
+        test.strictEqual(cli.program.option('')._version, '0.11.0', 'should list the program\'s version.');
+        test.strictEqual(cli.program.option('')._description, 'Lint the HTML of Bootstrap projects',
+            'should provide a description of Bootlint.');
+        test.strictEqual(cli.program.option('')._usage, '[options] [files...]', 'should provide Bootlint usage options.');
+        test.done();
+    },
+    'Version flag': function (test) {
+        test.expect(7);
+        test.strictEqual(cli.program.option('').options[0].flags, '-V, --version', 'should list the version flag');
+        test.strictEqual(cli.program.option('').options[0].required, 0, 'should not require the version flag.');
+        test.strictEqual(cli.program.option('').options[0].optional, 0, 'should return whether the version flag is optional.');
+        test.strictEqual(cli.program.option('').options[0].bool, true, 'should return whether the version flag is optional.');
+        test.strictEqual(cli.program.option('').options[0].short, '-V', 'should list the short version flag');
+        test.strictEqual(cli.program.option('').options[0].long, '--version', 'should list the long version flag');
+        test.strictEqual(cli.program.option('').options[0].description, 'output the version number',
+            'should provide a description of the version flag');
+        test.done();
+    },
+    'Disabled flag': function (test) {
+        test.expect(7);
+        test.strictEqual(cli.program.option('').options[1].flags, '-d, --disable <IDs>', 'should list the disable flag');
+        test.strictEqual(cli.program.option('').options[1].required, -15, 'should not require the disabled flag.');
+        test.strictEqual(cli.program.option('').options[1].optional, 0, 'should return whether the version flag is optional.');
+        test.strictEqual(cli.program.option('').options[1].bool, true, 'should return whether the version flag is optional.');
+        test.strictEqual(cli.program.option('').options[1].short, '-d', 'should list the short disable flag');
+        test.strictEqual(cli.program.option('').options[1].long, '--disable', 'should list the long disable flag');
+        test.strictEqual(cli.program.option('').options[1].description, 'Comma-separated list of disabled lint problem IDs',
+            'should provide a description of the disable flag');
+        test.done();
+    },
+    'Handle standard input': function (test) {
+        test.expect(6);
```

Comment:
Perhaps you could work with https://github.com/cjohansen/Sinon.JS and actually stub out `process.stdin`.